source/plugins/ruby/kubelet_utils.rb

```diff
@@ -17,11 +18,14 @@ def get_node_capacity
         cpu_capacity = 1.0
         memory_capacity = 1.0
 
-        response = CAdvisorMetricsAPIClient.getNodeCapacityFromCAdvisor(winNode: nil)
+        response = CAdvisorMetricsAPIClient.getAllMetricsCAdvisor(winNode: nil)
```

Comment:
these calls can also fail, probably checking for status code and also emitting telemetry is good idea.

Reply:
we are already emitting telemetry on failure here. (L33)

Reply:
> we are already emitting telemetry on failure here. (L33)

> @r-dilip, what I am saying,  extract metrics only when response.code == 200 and if its not 200, just emit the telemetry.  this we can avoid sending un-necessarily full call stack for all non-200 responses.